src/proxy.js

```diff
@@ -112,11 +107,11 @@
     const setter = handler.set ? function(prop, value) {
       throwRevoked('set');
       const status = handler.set(this, prop, value, proxy);
-      if (!status) {
+      // if (!status) {
```

Comment:
Commented the whole block. Closure compiler didn't like this empty block.

Reply:
I'm surprised Closure isn't now barfing because `status` is unused.

Remove the conditional and add it to the TODO? maybe...

```js
// TODO(samthor): If status is false and the calling code is in strict mode, throw TypeError.
// ...
```

Reply:
It does indeed complain about it.
![image](https://user-images.githubusercontent.com/11255103/37022092-8fd20a68-212a-11e8-8370-65c3b8d89e4d.png)

Moved the TODO up.